src/tests/functional/shared_test_classes/src/single_op/slice.cpp

```diff
@@ -59,6 +59,11 @@ void Slice8LayerTest::SetUp() {
         slice = std::make_shared<ov::op::v8::Slice>(param, begin_node, end_node, stride_node);
     }
     function = std::make_shared<ov::Model>(slice->outputs(), ov::ParameterVector{param}, "Slice-8");
+
+    // [PK]: Slice op only moves memory, no calculation takes places, so
+    // any difference likely means a bug.
+    rel_threshold = 0;
+    abs_threshold = 0;
```

Comment:
IMO it will be wrong. At this level we should only test the operator itself. If there are any implicit conversions - they should be disabled for test(how to do it?)- otherwise we might not catch the error in slice operator if values are close enough.

Reply:
For operator only testing unit tests should be used. 
Functional single layers tests run e2e plugin pipeline for trivial IRs using public API. There's no way to disable transformations (such as precision conversion) in that case.

Reply:
Ok, so will remove it completely, but just wanted to mention that default thresholds(which are calculated in runtime, depending on data values), are absurdly big - e.g. 1. for integer cases, e.g. ~0.2 for floating point, and to my best knowledge(asked people about it) - there is no easy way to set it per case.

Reply:
> Functional single layers tests run e2e plugin pipeline for trivial IRs using public API. There's no way to disable transformations (such as precision conversion) in that case.
In that case such reference tests should be moved to unit tests IMO. Of course nothing stands in the way to do such comparison tests at two levels - but higher level should focus on integration itself( with transformations, etc ), not on the validating operator itself. But again - it is outside the scope of this task to do such refactoring, so I will just remove setting that threshold.